test/integration/Drawing.e2e.test.js

```diff
@@ -0,0 +1,63 @@
+/// <reference types="Cypress" />
+describe('Drawing', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+
+    it('should create a new drawing annotation on a document', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty drawing layer to be present
+        cy.getByTestId('ba-Layer--drawing');
+
+        // Assert that the drawing creator does not exist and no annotations are present
+        cy.getByTestId('ba-DrawingCreator').should('not.exist');
+        cy.get('.ba-DrawingTarget').should('not.exist');
+
+        // Enter drawing creation mode
+        cy.getByTestId('bp-AnnotationsControls-drawBtn').click();
+
+        // Add a drawing annotation on the document
+        cy.drawRegion('ba-DrawingCreator');
```

Comment:
Seems like we can reuse the functionality but perhaps to make it less confusing we make have a generic utility and a separate method for `drawRegion` and `drawStroke`